lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1240,6 +1283,8 @@ def test_basic(self, mock_emit):
             "endorsed_comment_list_url": None,
             "non_endorsed_comment_list_url": None,
             "editable_fields": ["abuse_flagged", "following", "raw_body", "title", "topic_id", "type", "voted"],
+            'read': False,
+            'has_endorsed': False
```

Comment:
Some of them. `has_endorsed` and `endorsed` is what we call them, but its also what we get back so to keep it consistent, that's what we call them.